filament/backend/src/opengl/OpenGLProgram.cpp

```diff
@@ -46,14 +46,14 @@ struct OpenGLProgram::LazyInitializationData {
     Program::UniformBlockInfo uniformBlockInfo;
     Program::SamplerGroupInfo samplerGroupInfo;
     std::array<Program::UniformInfo, Program::UNIFORM_BINDING_COUNT> bindingUniformInfo;
+    utils::FixedCapacityVector<Program::PushConstant> pushConstants;
+    uint8_t pushConstantFragmentStageOffset;
```

Comment:
I think it's fine to do it this way, but I would prefer that lazyInitializationData just store both vectors (it's temporary, and efficient because you can std::move), and instead have the logic of packing them into a single vector done during the actual program initialization.